src/include/catalog/foreign_key.h

```diff
@@ -29,50 +29,48 @@ namespace catalog {
 // Stores info about foreign key constraints, like the sink table id etc.
 class ForeignKey {
  public:
```

Comment:
The reason we have strings for table and column names here is that the FK information is added to the CreatePlan for a table in the planner, at which point aren't passed anything but the strings that came in from the parser.